### ISSUE TITLE
TypeError: Class constructor Scp cannot be invoked without 'new'

### ISSUE THREAD

**Issue by Zhengyangzuishuai** [PantelisGeorgiadis/dcmjs-dimse#51]
I tried to use Scp in Electron like examples, the error is as indicated in the title.

**Comment by PantelisGeorgiadis** [comment]
Hello @Zhengyangzuishuai,
Thanks for reporting this! Unfortunately, I haven't tried to use the library in an electron project.
Do you have and can share the stack trace so we can understand where’s the root cause of the error?
How about instead of pulling the npm package, to use the source code files directly in your project?
This way you might be able to gather more details about the error and propose a possible fix.

Please let me know about the outcome!

**Comment by PantelisGeorgiadis** [comment]
Hi @Zhengyangzuishuai,
Although still far from an expert in electron, I used the app skeleton code found [here](https://github.com/fvorwerk/electron-skeleton) to create a basic SCP server. I updated the electron npm to the latest version, installed dcmjs-dimse and I just copy-pasted the SCP example from the README section. In no time, I had a working-as-expected C-ECHO SCP server. 

Are you still facing issues with your electron project?
